src/SingleCellAnnotator.imjoy.html

```diff
@@ -397,11 +397,11 @@
       imageData.cell_id = cell_id;
       let image_url = "/images/" + this.current_task_name + "/" + imageData.cell_id;
       if (extend_image){
-        image_url += "?extend=200&maskout=1";
-        imageData.extend = 200;
+        image_url += "?extend=500&maskout=1";
+        imageData.extend = 500;
       }else{
-        image_url += "?extend=20&maskout=1"
-        imageData.extend = 20;
+        image_url += "?extend=200&maskout=1"
+        imageData.extend = 200;
```

Comment:
Why do we extend so much? The issue of extending too much is that it will slow down the loading, a lot I guess.

Reply:
now all images is extend with 200. what is your suggestion for the value?

Reply:
it is caching to image_data_cached, so speed should be not slowed down, right?

Reply:
I don't know what's your speed, but on my side, the caching is very slow sometimes, I need to wait until I can click next.